apps/website/inertia/components/home/reasons_to_enroll_item.tsx

```diff
@@ -0,0 +1,21 @@
+import { JSX } from "react";
+
+interface ReasonsToEnrollItemProps {
+  icon: JSX.Element;
+  title: string;
+  description: string;
+}
+
+export default function ReasonsToEnrollItem({
+  icon,
+  title,
+  description,
+}: ReasonsToEnrollItemProps) {
+  return (
+    <div className="bg-enei-beige text-enei-blue mt-14 flex flex-col gap-y-3 p-12">
+      <div className="mx-auto h-10 w-10">{icon}</div>
+      <h3 className="text-center text-2xl font-bold">{title}</h3>
+      <p className="text-center">{description}</p>
```

Comment:
When the `description` text becomes slightly larger than the rest, the whole component can become slightly misaligned vertically: 
![image](https://github.com/user-attachments/assets/dbcfc937-e56f-4630-b095-0ef6aafc3d7c)

Personally I'm not a fan of this, and would suggest that all elements start at the same position vertically, taking up more space below if needed. A second opinion here would be appreciated.